styles/japanesewithanime.com/catppuccin.user.css

```diff
@@ -0,0 +1,397 @@
+	@green: @catppuccin[@@lookup][@green];
+	@teal: @catppuccin[@@lookup][@teal];
+	@sky: @catppuccin[@@lookup][@sky];
+	@sapphire: @catppuccin[@@lookup][@sapphire];
+	@blue: @catppuccin[@@lookup][@blue];
+	@lavender: @catppuccin[@@lookup][@lavender];
+	@text: @catppuccin[@@lookup][@text];
+	@subtext1: @catppuccin[@@lookup][@subtext1];
+	@subtext0: @catppuccin[@@lookup][@subtext0];
+	@overlay2: @catppuccin[@@lookup][@overlay2];
+	@overlay1: @catppuccin[@@lookup][@overlay1];
+	@overlay0: @catppuccin[@@lookup][@overlay0];
+	@surface2: @catppuccin[@@lookup][@surface2];
+	@surface1: @catppuccin[@@lookup][@surface1];
+	@surface0: @catppuccin[@@lookup][@surface0];
+	@base: @catppuccin[@@lookup][@base];
+	@mantle: @catppuccin[@@lookup][@mantle];
+	@crust: @catppuccin[@@lookup][@crust];
+	@accent-color: @catppuccin[@@lookup][@@accent];
+	@link-color: @catppuccin[@@lookup][@@lnk];
+
+
+
+  .set-colors() when not (@accent-color = @yellow) and not (@link-color = @yellow) {
+    @warn-color: @yellow;
+  }
+  .set-colors() when (@accent-color = @yellow) or (@link-color = @yellow) {
+    @warn-color: @peach;
+  }
+  .set-colors() when (@accent-color = @peach) and (@link-color = @yellow) {
+    @warn-color: @maroon;
+  }
+  .set-colors() when (@accent-color = @yellow) and (@link-color = @peach) {
+    @warn-color: @maroon;
+  }
+
+  .set-colors();
+
+
+
+
+	color-scheme: if(@lookup = latte, light, dark);
+
+	::selection {
+		background-color: fade(@accent-color, 30%);
+	}
+
+	input,
+	textarea {
+		&::placeholder {
+			color: @subtext0 !important;
+		}
+	}
+
+	body, html{
+		color: @text !important;
+		background-color: @base !important;
+	}
+
+
+
+  #megaNav__contents{
+		color: @text !important;
+		background-color: @surface1 !important;
+	}
+	.featured-posts i,
+	:not(a).transcript-extra{
+		color: @accent-color;
+	}
+
+	.post-body i{
+		color: @accent-color;
+	}
+	.post-body a i{
+		color: inherit !important;
+	}
+
+	a.placeholder-link{
+		color: @red !important;
+	}
+	a:link,
+	a:visited{
+		color: @link-color !important;
+	}
+
+
+	input.gsc-search-button{
+		color: @text !important;
+
+	}
+	input.gsc-input{
+		color: @text !important;
+		background-color: @surface0 !important;
+	}
+
+  & when (@ogBar = 1) {
+    input.gsc-search-button{
+      border-color: @overlay1 !important;
+      border-radius: 2px !important;
+      border-width: 1px !important;
+      padding: 1.8px 6.6px !important;
+      border-style: solid !important;
+      background-color: @surface1 !important;
+    }
+    input.gsc-input{
+      border-color: @surface1 !important;
+    }
+  }
+  & when (@ogBar = 0) {
+    input.gsc-search-button{
+      background-color: @surface1 !important;
+      border-color: @surface1 !important;
+      border-width: 1px !important;
+      border-style: solid !important;
+      border-radius: 1px !important;
+    }
+    input.gsc-input{
+      border-color: @surface0 !important;
+      border-radius: 1px !important;
+    }
+  }
```

Comment:
Just go with one search bar styling I think? Again, whichever better fits the website's original appearance.